cli/src/host.rs

```diff
@@ -135,11 +135,14 @@ impl win32::Host for EnvRef {
     }
 
     fn canonicalize(&self, path: &str) -> Result<String, u32> {
-        let path = windows_to_host_path(path);
-        match std::fs::canonicalize(path) {
-            Ok(ref p) => Ok(host_to_windows_path(p)),
-            Err(ref e) => Err(io_error_to_win32(e)),
+        let mut path = windows_to_host_path(path);
```

Comment:
It's becoming less clear to me what this canonicalize function is supposed to be doing.  Seems to convert relative paths to absolute too (?)

Reply:
Yeah, this has evolved into just a "normalize path" function. I'll rework this to live on the win32 side instead. We'll just need to keep track of the current working directory in kernel32.

Reply:
Reworked the host interface, it now just provides a `current_dir` and the rest is handled on the win32 side.